src/app/app.component.html

```diff
@@ -1 +1,3 @@
-<div>App</div>
+<bs-navbar></bs-navbar>
+
+<main role="main" class="container"></main>
```

Comment:
do you need this line?

Reply:
Are you referring to line 1:  `<div>App</div>` ?

He removes this line when he extracts the HTML in this file to the Navbar component.  I haven't seen a need for it since.